templates/exadata-limits/main.tf

```diff
@@ -0,0 +1,24 @@
+locals {
+
+  output_zones     = length(local.zone_mappings) == 0 ? "N/A" : var.zones
+  zone_mappings    = join("\n", distinct(flatten([
+    for k in flatten([for z in module.zone_mappings : z.location_zone_mapping]) :
+    "\nlogical_zone: ${k.logical_zone}, physical_zone: ${k.physical_zone}"
+  ])))
+  shape_per_zone = join("\n", flatten([for z in module.zone_mappings : "AZ logical Zone: ${z.logical_zone} AZ Pyhsical Zone: ${z.az_physical_zone} Internal AD: ${z.internal_ad}:\n${local.shape_info}\n"]))
```

Comment:
Can we use templatefile here https://developer.hashicorp.com/terraform/language/functions/templatefile